src/simularium/types.ts

```diff
@@ -77,7 +77,10 @@ export interface TrajectoryFileInfoV2 extends TrajectoryFileInfoBase {
     };
 }
 
-export type TrajectoryFileInfo = TrajectoryFileInfoV1 | TrajectoryFileInfoV2;
+export type TrajectoryFileInfoAny = TrajectoryFileInfoV1 | TrajectoryFileInfoV2;
+
+// This should always point to the latest version
+export type TrajectoryFileInfo = TrajectoryFileInfoV2;
```

Comment:
I wonder if we can make `version` a static readonly member of the data structure something like:
TrajectoryFileInfoV1: {
...
static readonly version: number = 2;
...
}

Reply:
You mean in the type definition right? I can't initialize it to a specific number, but I can stick a `readonly` in front of `version: number;` under `TrajectoryFileInfoBase` if that sounds good?